pkg/parse/parse.go

```diff
@@ -510,6 +510,70 @@ func AuthConfig(creds string) (*types.DockerAuthConfig, error) {
 	}, nil
 }
 
+// GetBuildOutput is responsible for parsing custom build output argument i.e `build --output` flag.
+// Takes `buildOutput` as string and returns BuildOutputOptions
+func GetBuildOutput(buildOutput string) (define.BuildOutputOption, error) {
+	isDir := true
+	isStdout := false
+	typeSelected := false
+	pathSelected := false
+	path := ""
```

Comment:
(Absolutely non-blocking: All of this can now be moved just before the `tokens :=` initializer, requiring a tiny bit less context to keep in mind when reading the code.)